### PR TITLE
Dynamic load tree

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: MediaModelBundle/Repository/FolderRepository.php [hunk 1]
```diff
@@ -16,11 +16,14 @@ class FolderRepository extends DocumentRepository implements FolderRepositoryInt
      *
      * @return Collection
      */
-    public function findAllRootFolder($siteId = null)
+    public function findAllFolder($siteId = null, $parentId = null)
```
Comment (itkg-alavieille) [inline MediaModelBundle/Repository/FolderRepository.php]: add the parameters in interface

Comment (itkg-nanne) [inline MediaModelBundle/Repository/FolderRepository.php]: done.

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**